ducktape/cluster/vagrant.py

```diff
@@ -12,9 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .cluster import Cluster, ClusterSlot
-from .json import JsonCluster
+from __future__ import absolute_import
 
+from .json import JsonCluster
+from .cluster import Cluster, ClusterSlot
+from ducktape.command_line.config import ConsoleConfig
+import os
+import json
```

Comment:
I ran into the same issue when prototyping some new code. I don't think we should rename since the absolute import is required in the future (3.0+) anyway. In the few places we need it, we can just use the `from __future__` import.